client/src/fakeData/persona.js

```diff
@@ -0,0 +1,23 @@
+module.exports = [
+  {
+    id: '1',
+    img: 'https://i.imgur.com/ifAbUdp.png',
+    title: 'Mark',
+    text:
+      'Is a single mun with a young baby.She would like to access theenternet but thinks the connections and equipment would be tooexpensiv ',
```

Comment:
Need a space between "baby." and "She"
Also, later in the line should read 'the Internet' instead of 'theenterenet'
and 'too expensive.' instead of 'tooexpensiv'